t/test.pl

```diff
@@ -476,6 +476,17 @@ ($$$@)
     _ok($pass, _where(), $name, @mess);
 }
 
+sub refcount_is {
+    my (undef, $expected, $name, @mess) = @_;
```

Comment:
IIRC our policy is that `t/test.pl` is strictly for the use of the core distribution and is therefore **not** documented with POD.  But perhaps some inline comments would be helpful here.  In particular, could you explain why the first argument to a call of this function is set to `undef`?

Reply:
Ahyes; that's fair. I'll pop something in before I merge it.